cli/replicasets/common.go

```diff
@@ -0,0 +1,185 @@
+package replicasets
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"strings"
+
+	"github.com/apex/log"
+	"github.com/tarantool/cartridge-cli/cli/common"
+	"github.com/tarantool/cartridge-cli/cli/context"
+	"github.com/tarantool/cartridge-cli/cli/project"
+	"github.com/tarantool/cartridge-cli/cli/running"
+	"gopkg.in/yaml.v2"
+)
+
+type InstanceConf struct {
+	URI string `yaml:"advertise_uri"`
+}
+
+type InstancesConf map[string]*InstanceConf
+
+func getInstancesConf(ctx *context.Ctx) (*InstancesConf, error) {
+	var err error
+
+	log.Debugf("Instances configuration file is %s", ctx.Running.RunDir)
+
+	if _, err := os.Stat(ctx.Running.ConfPath); err != nil {
+		return nil, fmt.Errorf("Failed to use instances configuration file: %s", err)
+	}
+
+	fileContentBytes, err := common.GetFileContentBytes(ctx.Running.ConfPath)
+	if err != nil {
+		return nil, fmt.Errorf("Failed to read instances configuration file: %s", err)
+	}
+
+	var allSectionsConf InstancesConf
+	if err := yaml.Unmarshal([]byte(fileContentBytes), &allSectionsConf); err != nil {
+		return nil, fmt.Errorf("Failed to parse instances configuration file %s: %s", ctx.Running.ConfPath, err)
+	}
+
+	instancesConf := make(InstancesConf)
+
+	appInstancePrefix := fmt.Sprintf("%s.", ctx.Project.Name)
+	for key, sectionConf := range allSectionsConf {
```

Comment:
Should we ignore/consider stateboard here and at https://github.com/tarantool/cartridge-cli/pull/408/files#diff-4fb23dc0ae21422ebe8be6bd721532279e77e346e549a5294f020b4c7443fa62R285 ?

Reply:
Stateboard isn't a part of topology, so I think it should be ignored.
`common. GetInstancesFromArgs` function is also used for starting/stopping. Stateboard isn't passed as an argument, it should be specified by flag.

Reply:
Thanks for explanation